components/dropbox/actions/download-file-to-tmp/download-file-to-tmp.mjs

```diff
@@ -0,0 +1,46 @@
+import dropbox from "../../dropbox.app.mjs";
+import common from "../common/common.mjs";
+import fs from "fs";
+import { file } from "tmp-promise";
+
+export default {
+  ...common,
+  name: "Download File To TMP",
+  description: "Download a specific file to the temporary directory. [See docs here](https://dropbox.github.io/dropbox-sdk-js/Dropbox.html#filesDownload__anchor)",
+  key: "dropbox-download-file-to-tmp",
+  version: "0.0.11",
+  type: "action",
+  props: {
+    dropbox,
+    path: {
+      propDefinition: [
+        dropbox,
+        "pathFolder",
+      ],
+      description: "The file path in the user's Dropbox to download.",
+    },
+    name: {
+      type: "string",
```

Comment:
This prop should be optional, then the user can decide either change or not the current file name